cli/jobs/single-step/gpu_perf/README.md

```diff
@@ -0,0 +1,126 @@
+---
+page_type: sample
+languages:
+- azurecli
+- shell
+- docker
+products:
+- azure-machine-learning
+description: This sample shows how to run standard health checks on a GPU cluster to test performance and correctness of distributed multinode GPU jobs. It helps with troubleshooting performance issues related to the environment and container.
+---
+
+# Run NCCL tests on GPU to check performance and configuration
+
+This job will run [NCCL test](https://github.com/NVIDIA/nccl-tests) checking performance and correctness of NCCL operations on a GPU node. It will also run a couple of standard tools for troubleshooting (nvcc, lspci, etc).
+
+The goal here is to verify the performance of the node and availability in your container of the drivers, libraries, necessary to run optimal distributed gpu jobs.
+
+## How to
+
+### Run out of the box
+
+This will use a local definition of the environment that you can use to iterate on your container design.
+
+```bash
+# run the job
+az ml job create  -f ./gpu_perf_job.yml --web
+```
+
+To modify the settings of the job, you can either modify the yaml, or override from the command line.
+
+In particular, in `gpu_perf_job.yml`, please check the following:
+- to run on a specific compute, set `compute=azureml:name`
+- to adapt to the number of gpus on the node, set `distribution.process_count_per_instance=N`
+- for multi-node, set `resources.instance_count=N`
+
+### Use registered environments
+
+You can pre-register each environment in your workspace and override the job to use those instead:
+
+```bash
+# create the environments
+az ml environment create  --file ./environments/azureml/env.yml
+az ml environment create  --file ./environments/nvidia/env.yml
+
+# run the job and manually override its environment
+az ml job create -f ./gpu_perf_job.yml --web --set environment="azureml:nccltests_azureml:openmpi4.1.0-cuda11.1-cudnn8-ubuntu18.04"
+```
+
+## How to customize
+
+To check perf against your own container/config:
+
+1. Create an environment based on the content from directory `environments/azureml/`.
+
+2. Create this environment using `az ml environment create` command above.
+
+3. Modify `gpu_perf_job.yml` to use your new environment name/version.
+
+4. Run the job using `az ml job create`.
+
+## Set environment variables
+
+In `gpu_perf_job.yml` you'll find an environment variables section that you can leverage for testing your specific configuration.
+
+For examples please see:
+- specs of [UCX environment variables](https://rocmdocs.amd.com/en/latest/Remote_Device_Programming/UCXenv.html)
```

Comment:
remove locale in URL if possible. also these types of links tend to break over time, would recommend not having them (or doing the work in this repo to validate links -- docs does this)

Reply:
Good point. Sadly I checked the url and removing the locale doesn't work.